server/chat/backend/agent/tools/thousandeyes_tool.py

```
@@ -0,0 +1,554 @@
+        return False, _ERROR_USER
+
+    client = _build_client(user_id)
+    if not client:
+        return False, _ERROR_NOT_CONNECTED
+
+    try:
+        data = api_fn(client)
+        return True, data
+    except ThousandEyesAPIError as exc:
+        return False, json.dumps({"error": f"ThousandEyes API error: {exc}"})
+    except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)